app/policies/policy_context.rb

```diff
@@ -0,0 +1,12 @@
+class PolicyContext
+  attr_reader :record
+
+  def initialize(record, user)
+    @record = record
+    @user = user
+  end
+
+  def policy_class
+    "#{@user.roles.uniq.first}DefenceRequestPolicy".classify
```

Comment:
This is ok for the time being.
It will need to change in the near future once we decide how to handle the "super" or "admin" version of each cso, cco and solicitor role